px-deck-selector.html

```diff
@@ -33,18 +33,18 @@
   <template>
     <style include="px-deck-selector-styles"></style>
 
-    <div class$="dropdown-text actionable actionable--action {{_hideChevronIfNoDecks(_isDecks)}}" on-click="_toggleDropdown">
+    <div id="trigger" class$="dropdown-text actionable actionable--action [[_hideChevronIfNoDecks(decks)]]" on-click="_handleTriggerTapped">
       <span class="truncate" title={{selectedDeck.name}}>{{selectedDeck.name}}</span>
       <px-icon icon="px-utl:chevron"></px-icon>
     </div>
-    <ul id="dropdown" class="dropdown-menu list-bare visuallyhidden">
+    <ul id="dropdown" class$="dropdown-menu list-bare [[_getDropdownClassName(_isDropdownOpen)]]">
       <template is="dom-repeat" items="{{decks}}" as="deck">
-        <li on-click="_selectDeck" class$="dropdown-menu__item {{_isSelected(deck,selectedDeck)}}">
```

Comment:
should we use on-tap rather than on-click?

Reply:
👍